libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
+        properties[field["name"]] = {
+            "type": "string",
+            "index": "analyzed"
+        }
+    elif field.get("type") == "long":
+        properties[field["name"]] = {
+            "type": "long",
+            "doc_values": "true"
```

Comment:
Another thing that I noticed is that `doc_values` is true by default, so no need to add it explicitly everywhere. This is my mistake :)